stats/admin.py

```diff
@@ -53,6 +57,8 @@ def run_mds(self, request, obj):
         except ValueError:
             message = 'Something went wrong while running scenario {}'.format(obj.title)
             self.message_user(request, message, level=messages.ERROR)
+            logger.exception('Error running scenario {}'.format(obj.title))
```

Comment:
Where is this logged to?

Reply:
I think it depends on how Django is configured. I added it because it helped me debug my changes (and when running the development sever it outputs to the console)